ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -286,6 +286,8 @@ def _create_session_core(
         self,
         session_constructor: Callable[[SessionInformation], TSession],
         instrument_type_id: str,
+        initialization_behavior: SessionInitializationBehavior = SessionInitializationBehavior.AUTO,
+        can_emulate: bool = True,
```

Comment:
Replace with a callable as described in the other thread.

Reply:
Updated.